src/PKSim.Core/Snapshots/PopulationAnalysisField.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Drawing;
+using OSPSuite.Core.Domain;
+using PKSim.Core.Model.PopulationAnalyses;
+
+namespace PKSim.Core.Snapshots
+{
+   public class PopulationAnalysisField : IWithName, IWithDescription
```

Comment:
Simple flatten snapshot. The model objects are very complicated and we have different object for each use cases. As snapshot, stuff that does not matter is set to null and will not be saved to json. Simple and efficient